sycamore/tests/unit/transforms/test_regex_replace.py

```diff
@@ -52,3 +61,13 @@ def test_via_execute(self, mocker):
     def validateElems(self, elems: list[Element]):
         assert len(elems) == 1
         assert elems[0].text_representation == "foo bar"
+
+    def expectValid(self, spec):
+        RegexReplace(None, spec)
+
+    def expectInvalid(self, spec):
+        try:
+            RegexReplace(None, spec)
+            raise AssertionError("RegexReplace accepted bad spec")
+        except Exception:
```

Comment:
make sure to try to catch the error you expect.
except RuntimeError:
optionally 
except RuntimeError as r:
  if r.args[0] != "RegexReplace spec is not list[tuple[str, str]]":
     raise AssertionError("Wrong runtime error")